.changeset/twelve-cheetahs-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
```

Comment:
Just realized this is a breaking change. Major upgrade because of the change in our token shape.

Reply:
Updated

```json
"text": "#171717",
```

Previous

```json
"text": { "r": 23, "g": 23, "b": 23, "a": 1, "hex": "#171717" },
```

Reply:
Updated to major